aspnetcore/tutorials/razor-pages/da1.md

```diff
@@ -13,83 +13,97 @@ By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-The **Edit**, **Details**, and **Delete** links are generated by the [Anchor Tag Helper](xref:mvc/views/tag-helpers/builtin-th/anchor-tag-helper) 
-in the *Pages/Movies/Index.cshtml* file.
+   ![Browser window with mouse over the Edit link and a link Url of http://localhost:1234/Movies/Edit/5 is shown](~/tutorials/razor-pages/da1/edit7.png)
 
-[!code-cshtml[](~/tutorials/razor-pages/razor-pages-start/snapshot_sample/RazorPagesMovie/Pages/Movies/Index.cshtml?highlight=16-18&range=32-)]
+   The **Edit**, **Details**, and **Delete** links are generated by the [Anchor Tag Helper](xref:mvc/views/tag-helpers/builtin-th/anchor-tag-helper) in the *Pages/Movies/Index.cshtml* file.
 
-[Tag Helpers](xref:mvc/views/tag-helpers/intro) enable server-side code to participate in creating and rendering HTML elements in Razor files. In the preceding code, the `AnchorTagHelper` dynamically generates the HTML `href` attribute value from the Razor Page (the route is relative), the `asp-page`,  and the route id (`asp-route-id`). See [URL generation for Pages](xref:razor-pages/index#url-generation-for-pages) for more information.
+   [!code-cshtml[](~/tutorials/razor-pages/razor-pages-start/snapshot_sample/RazorPagesMovie/Pages/Movies/Index.cshtml?highlight=16-18&range=32-)]
 
-Use **View Source** from your favorite browser to examine the generated markup. A portion of the generated HTML is shown below:
+   [Tag Helpers](xref:mvc/views/tag-helpers/intro) enable server-side code to participate in creating and rendering HTML elements in Razor files.
 
-```html
-<td>
-  <a href="/Movies/Edit?id=1">Edit</a> |
-  <a href="/Movies/Details?id=1">Details</a> |
-  <a href="/Movies/Delete?id=1">Delete</a>
-</td>
-```
+   In the preceding code, the `AnchorTagHelper` dynamically generates the HTML `href` attribute value from the Razor Page (the route is relative), the `asp-page`,  and the route id (`asp-route-id`). See [URL generation for Pages](xref:razor-pages/index#url-generation-for-pages) for more information.
 
-The dynamically-generated links pass the movie ID with a query string (for example, the `?id=1` in  `https://localhost:5001/Movies/Details?id=1`).
+1. Use **View Source** from a favorite browser to examine the generated markup. A portion of the generated HTML is shown below:
 
-### Add route template
+   ```html
+   <td>
+     <a href="/Movies/Edit?id=1">Edit</a> |
+     <a href="/Movies/Details?id=1">Details</a> |
+     <a href="/Movies/Delete?id=1">Delete</a>
+   </td>
+   ```
 
-Update the Edit, Details, and Delete Razor Pages to use the "{id:int}" route template. Change the page directive for each of these pages from `@page` to `@page "{id:int}"`. Run the app and then view source. The generated HTML adds the ID to the path portion of the URL:
+   The dynamically-generated links pass the movie ID with a query string. For example, the `?id=1` in  `https://localhost:5001/Movies/Details?id=1`.
```

Comment:
As a rule of thumb, don't use a hyphen after words ending in "ly". I learned that one from Maira. 😄 

```suggestion
   The dynamically generated links pass the movie ID with a query string. For example, the `?id=1` in  `https://localhost:5001/Movies/Details?id=1`.
```